src/_cffi_src/openssl/bignum.py

```diff
@@ -72,6 +72,12 @@
 int BN_num_bytes(const BIGNUM *);
 
 int BN_mod(BIGNUM *, const BIGNUM *, const BIGNUM *, BN_CTX *);
+
+/* The following 3 prime methods are exposed for Tribler. */
+int BN_generate_prime_ex(BIGNUM *, int, int, const BIGNUM *,
+                         const BIGNUM *, BN_GENCB *);
+int BN_is_prime_ex(const BIGNUM *, int, BN_CTX *, BN_GENCB *);
+static const int BN_prime_checks_for_size(int);
```

Comment:
Any reason you chose to declare this as static?

Reply:
That's what the documentation (https://cryptography.io/en/latest/development/c-bindings/) gave as the format for `#define`'s. I'll change it to non-static then.

Reply:
Ah, yeah. The static keyword doesn't hurt here so it's fine, I was just curious why. Probably should update our docs to note that static makes sense for constants and not for macros though.